js/components/graph/FocusBar.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from "prop-types";
+import FocusTab from "./FocusTab.js";
+
+// These lists are in reverse order to what ends up appearing on the screen
+const computerScienceFocusLabels = [
+  ["web", "Web Technologies"],
+  ["theory", "Theory of Computation"],
+  ["HCI", "HumanComp Interaction"],
```

Comment:
Change this title to `Human-Computer Interaction`. If this causes the text to overflow, you should look into CSS rules for making the text automatically fit; if this is too tricky, though, we can save this for a future PR.